apps/betterangels-backend/accounts/admin.py

```diff
@@ -17,6 +21,28 @@
 )
 
 
+class NotesByAdminMixin(object):
+    def notes_by(self, obj: User) -> SafeString:
+        notes = Note.objects.filter(created_by=obj)
+        note_links = [
+            '<a href="{}">{}</a>'.format(reverse("admin:notes_note_change", args=(note.id,)), note.label_with_client)
+            for note in notes
+        ]
+        return format_html("<br>".join(note_links))
+
+
+class NotesForAdminMixin(object):
+    def notes_for(self, obj: User) -> SafeString:
+        notes = Note.objects.filter(client=obj)
+        note_links = [
+            '<a href="{}">{}</a>'.format(
+                reverse("admin:notes_note_change", args=(note.id,)), note.label_with_created_by
+            )
+            for note in notes
+        ]
+        return format_html("<br>".join(note_links))
+
+
```

Comment:
Is this being reused anywhere else besides here? I'm trying to understand why we went with a mixin instead of just adding it directly to the admin.

Reply:
Also I don't think this should be in accounts this seems to be note specific.

Reply:
delete this

Reply:
let's just nix this